packages/avatar/src/Avatar/Avatar.types.ts

```diff
@@ -0,0 +1,106 @@
+import { ComponentProps } from 'react';
+import omit from 'lodash/omit';
+
+import { GlyphName } from '@leafygreen-ui/icon';
+import { DarkModeProps, Theme } from '@leafygreen-ui/lib';
+import { Size } from '@leafygreen-ui/tokens';
+
+export const Format = {
+  /** Renders a MongoDB logo mark */
+  MongoDB: 'mongodb',
+
+  /** Renders the user's given name initial */
+  Text: 'text',
+
+  /** Renders a `Person` icon */
+  Icon: 'icon',
+
+  /** TODO: Renders an image avatar */
+  // Image: 'image',
+};
+export type Format = (typeof Format)[keyof typeof Format];
+
+export const AvatarSize = {
+  ...omit(Size, ['XSmall', 'Small']),
+  XLarge: 'xlarge',
+} as const;
+export type AvatarSize = (typeof AvatarSize)[keyof typeof AvatarSize];
+
+export interface BaseAvatarProps extends ComponentProps<'div'>, DarkModeProps {
+  /**
+   * The relative Size of tha Avatar
+   *
+   * @default AvatarSize.Default
+   */
+  size?: AvatarSize;
+
+  /** Renders that Avatar at a unique size, not supported by {@link AvatarSize} */
```

Comment:
what does `not supported by {@link AvatarSize}` mean?

Reply:
Any explicit px size that's not mapped to the `AvatarSize` enum